zulipterminal/model.py

```diff
@@ -858,6 +858,11 @@ def _handle_typing_event(self, event: Event) -> None:
         Handle typing notifications (in private messages)
         """
         assert event['type'] == "typing"
+
+        # Restore the footer if there hasn't been a corresponding typing
```

Comment:
This comment doesn't seem to belong here. You could add it prior to the next change.